client/src_analyzer/Worker_Analyze.re

```diff
@@ -105,4 +106,65 @@ module Make = (ESig: Worker_Evaluator.EvaluatorSig) => {
         };
       loop(codeBlocks, []);
     };
+
+  let logRefmtError = (code, error, targetLang) => {
+    let fromLang = ref("RE");
+    let toLang = ref("ML");
+
+    switch (targetLang) {
+    | ML => ()
+    | RE =>
+      fromLang := "ML";
+      toLang := "RE";
+    };
+
+    let message = {j|There was a problem reformating your $fromLang code to $toLang|j};
+    /* Notify.error(message ++ ". Check the console for details."); */
+    Js.log(message ++ "\n");
+    Js.log2("Code:\n", code);
+    Js.log2("Error:\n", error);
```

Comment:
currently showing the error only in the console, along with the offending code, here (and saying to check the console for more details), since I thought showing the error in the notification might be too much.

Do you want the error shown in the error toast as well?

Reply:
I think let's keep it there. Ideally, you would show the syntax error in the editor but let's leave it there for a moment :D

Reply:
👍

Reply:
whoops, I also just realized that I pulled all this code out into Worker_Refmt and never deleted it in Worker_Analyze. Will fix that in next commit.